packages/react/src/intl/BaseFormattedTime.tsx

```diff
@@ -14,7 +14,7 @@ export function BaseFormattedTime({
 
     return (
         <time {...attributes} dateTime={dateTime.toISOString()}>
-            {formatTime(dateTime, { locale, timezone })}
+            {formatTime(dateTime, { locale, timezone }).replace('GMT', 'UTC')}
```

Comment:
Do you think there's a better way to do this?

Reply:
I am not aware of a better way. This is good enough :)